tflite2onnx/op/logistic.py

```diff
@@ -0,0 +1,41 @@
+import logging
```

Comment:
Btw, we can include activation operators that don't have attributes in one converter class in https://github.com/jackwish/tflite2onnx/blob/master/tflite2onnx/op/activation.py,  but let me handle that.